src/ssl.c

```diff
@@ -34814,14 +34817,33 @@ int wolfSSL_ECDH_compute_key(void *out, size_t outlen,
     }
 
     len = (word32)outlen;
+    key = (ecc_key*)ecdh->internal;
+
+#if defined(ECC_TIMING_RESISTANT) && !defined(HAVE_SELFTEST) \
+    && !defined(HAVE_FIPS)
+    if (key->rng == NULL) {
```

Comment:
This does not look correct. What about the case case with key->rng != NULL, where is key->rng set then?

Reply:
`key->rng` should be set by wc_ecc_set_rng if using ECC_TIMING_RESISTANT. Would you like me to move this to an initialization of key (ex: wolfSSL_EC_KEY_new)?